File: Python/BOJ/배열/1475.py
arr = list(input())
dict = {str(n): 0 for n in range(10)}

for num in arr:
  if num == "6" or num == "9":
    if dict["6"] <= dict["9"]:
      dict["6"] += 1
    else:
      dict["9"] += 1
  else:
    dict[num] += 1

print(max(dict.values()))

File: Swift/BOJ/문제집/배열/1475.swift
//
//  1475.swift
//  Algorithm
//
//  Created by 유정주 on 2023/06/26.
//

import Foundation

let n = readLine()!.map { Int(String($0))! }
var arr: [Int] = Array(repeating: 0, count: 11)

for num in n {
    if num == 6 || num == 9 {
        if arr[6] <= arr[9] {
            arr[6] += 1
        } else {
            arr[9] += 1
        }
    } else {
        arr[num] += 1
    }
}

print(arr.max()!)
